apps/ui/src/stores/votingPowers.ts

```diff
@@ -111,9 +112,7 @@ export const useVotingPowersStore = defineStore('votingPowers', () => {
           0
         );
 
-        vpItem.canPropose =
-          totalProposeVp >= BigInt(item.proposal_threshold) ||
-          isSpaceMember(space as Space, account);
+        vpItem.canPropose ||= totalProposeVp >= BigInt(item.proposal_threshold);
```

Comment:
```suggestion
        vpItem.canPropose = totalProposeVp >= BigInt(item.proposal_threshold);
```

This line will be only called for non-members so propositionPower should be the only criterion.